flycheck.el

```diff
@@ -8788,6 +8789,26 @@ See URL `https://docs.python.org/3.4/library/py_compile.html'."
           line ", " column ", " (one-or-more not-newline) line-end))
   :modes python-mode)
 
+(flycheck-def-args-var flycheck-python-mypy-args python-mypy)
```

Comment:
We usually avoid generic `-args` variables, in favor of specific variables for each relevant flag; I think it's be better to do this here too.

Reply:
Which do you think are the most relevant?

```
$ mypy --help
usage: mypy [-h] [-v] [-V] [--python-version x.y] [--platform PLATFORM] [-2]
            [--ignore-missing-imports]
            [--follow-imports {normal,silent,skip,error}]
            [--disallow-any {unimported, expr, unannotated, decorated, explicit, generics}]
            [--disallow-untyped-calls] [--disallow-untyped-defs]
            [--disallow-incomplete-defs] [--check-untyped-defs]
            [--disallow-subclassing-any] [--warn-incomplete-stub]
            [--disallow-untyped-decorators] [--warn-redundant-casts]
            [--no-warn-no-return] [--warn-return-any] [--warn-unused-ignores]
            [--warn-unused-configs] [--show-error-context]
            [--no-implicit-optional] [-i] [--quick-and-dirty]
            [--cache-dir DIR] [--skip-version-check] [--strict-optional]
            [--strict-optional-whitelist [GLOB [GLOB ...]]]
            [--junit-xml JUNIT_XML] [--pdb] [--show-traceback] [--stats]
            [--inferstats] [--custom-typing MODULE]
            [--custom-typeshed-dir DIR] [--scripts-are-modules]
            [--config-file CONFIG_FILE] [--show-column-numbers]
            [--find-occurrences CLASS.MEMBER] [--strict]
            [--shadow-file SOURCE_FILE SHADOW_FILE] [--any-exprs-report DIR]
            [--cobertura-xml-report DIR] [--html-report DIR]
            [--linecount-report DIR] [--linecoverage-report DIR]
            [--memory-xml-report DIR] [--old-html-report DIR]
            [--txt-report DIR] [--xml-report DIR] [--xslt-html-report DIR]
            [--xslt-txt-report DIR] [-m MODULE] [-c PROGRAM_TEXT] [-p PACKAGE]
            [files [files ...]]
```

I like to run with `--py2 --strict --disallow-any=generics` for new scripts, or `--py2 --check-untyped-defs` when I'm gradually porting a codebase (to check calls to annotated functions even when the caller isn't yet annotated).

`--strict` includes quite a few arguments -- do we offer the individual arguments too?

Reply:
My usual approach is to add whatever the original writer of the checker finds useful, and accept pull requests for more as needed :)